com.unity.ml-agents/Runtime/SideChannel/FloatPropertiesChannel.cs

```diff
@@ -42,9 +42,9 @@ public class FloatPropertiesChannel : SideChannel, IFloatProperties
         Dictionary<string, float> m_FloatProperties = new Dictionary<string, float>();
         Dictionary<string, Action<float>> m_RegisteredActions = new Dictionary<string, Action<float>>();
 
-        public override int ChannelType()
+        public FloatPropertiesChannel()
```

Comment:
I think this (and maybe EngineConfigurationChannel, but that's debatable) should default to `ReservedChannelId.FloatProperties` but allow overriding in the constructor. So either overloaded constructors, or an int arg with a default value.